product_multi_image/test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo14-addon-base-multi-image @ git+https://github.com/hailangvn/server-tools@14.0-mig-base_multi_image#subdirectory=setup/base_multi_image
```

Comment:
whandering what is that?

Reply:
No idea, I thought it was inside `product_multi_image` from  hailangvn, but I checked again and turns out it is not. Removed :)